agr_java_core/src/main/java/org/alliancegenome/neo4j/entity/SpeciesType.java

```diff
@@ -170,6 +170,10 @@ public static String getTaxonId(String species) {
 		return species;
 	}
 
+	// Fix up species name with yeast correction
+	public static String getSpeciesNameCorrected(String name){
```

Comment:
Given that this is yeast only, I would suggest a different name for this method.

Reply:
I want to keep this method generic, in case other species do something similar.